bundles/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/handler/CBusCGateHandler.java

```diff
@@ -406,6 +357,10 @@ public void processStatusChange(@Nullable CGateSession cGateSession, @Nullable S
                         commentTokenizer.poll();
                         @Nullable
                         String commentToken = commentTokenizer.peek();
+                        /// var/log/openhab2/openhab.log.7:2020-04-05 22:09:47.786 [DEBUG]
```

Comment:
Was this an intended change?

Reply:
No i just pushed another change removing it. Sorry for leaving that in and thanks for spotting it.